AWSLex/Bluefront/include/BFAudioSource.h

```diff
@@ -71,13 +71,20 @@ voiceActivityDetectorConfiguration:(BFVADConfig *)vadConfig;
 /**
  * Marks the BFAudioSource as having stopped and stops any recording
  * processes required underneath.
- */
+*/
```

Comment:
nittiest of nits: This was right the first time :)
```suggestion
 */
```